lucille-core/src/main/java/com/kmwllc/lucille/connector/BlankConnector.java

```diff
@@ -0,0 +1,36 @@
+package com.kmwllc.lucille.connector;
+
+import com.kmwllc.lucille.core.ConnectorException;
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Publisher;
+import com.typesafe.config.Config;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Connector implementation that produces blank documents given amount to produce
+ */
+public class BlankConnector extends AbstractConnector {
+
+  private static final Logger log = LoggerFactory.getLogger(BlankConnector.class);
+  private final int docNum;
+  private final int docStartNum;
```

Comment:
could rename to startWith or sequenceMinValue

Reply:
done